source/uwp/Renderer/lib/ActionHelpers.cpp

```diff
@@ -677,21 +676,44 @@ namespace AdaptiveNamespace::ActionHelpers
 
         if (action != nullptr)
         {
-            // If we have an action, use the title for the AutomationProperties.Name
+            // If we have an action, use the title for the AutomationProperties.Name and tooltip
             HString title;
             THROW_IF_FAILED(action->get_Title(title.GetAddressOf()));
 
-            ComPtr<IDependencyObject> buttonAsDependencyObject;
-            THROW_IF_FAILED(button.As(&buttonAsDependencyObject));
+            if (title.IsValid())
+            {
+                // Set the automation properties name
+                ComPtr<IDependencyObject> buttonAsDependencyObject;
+                THROW_IF_FAILED(button.As(&buttonAsDependencyObject));
+
+                ComPtr<IAutomationPropertiesStatics> automationPropertiesStatics;
+                THROW_IF_FAILED(
+                    GetActivationFactory(HStringReference(RuntimeClass_Windows_UI_Xaml_Automation_AutomationProperties).Get(),
+                                         &automationPropertiesStatics));
 
-            ComPtr<IAutomationPropertiesStatics> automationPropertiesStatics;
-            THROW_IF_FAILED(
-                GetActivationFactory(HStringReference(RuntimeClass_Windows_UI_Xaml_Automation_AutomationProperties).Get(),
-                                     &automationPropertiesStatics));
+                THROW_IF_FAILED(automationPropertiesStatics->SetName(buttonAsDependencyObject.Get(), title.Get()));
 
-            THROW_IF_FAILED(automationPropertiesStatics->SetName(buttonAsDependencyObject.Get(), title.Get()));
+                WireButtonClickToAction(button.Get(), action, renderContext);
 
-            WireButtonClickToAction(button.Get(), action, renderContext);
+                // Also use the title as the tooltip
+                ComPtr<IToolTip> toolTip =
```

Comment:
Are there any accessibility settings/configuration needed around tooltips? Also is there any way to suppress tooltips instead of always displaying it? Based on recommendations here [https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tooltips#recommendations](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/tooltips#recommendations) and in general Accessibility concerns looks like tooltips (like placeholders) could be prone to abuse/misuse etc ?

Reply:
This scenario is using the title as a tooltip on a select action. As you can see in the code above, we're already using that title as the accessibility text for the button that is the select action. This tooltip is designed to provide more information to sighted users as to what will happen when they click the select action. For suppression, were you thinking for the card author or the host? In what scenario would they want to suppress it? 

---
In reply to: [523179028](https://github.com/microsoft/AdaptiveCards/pull/5044#discussion_r523179028) [](ancestors = 523179028)

Reply:
For what it's worth on the accessibility question, i just tested it out using keyboard nav and narrator. When you navigate to the select action using keyboard navigation the tooltip does show, and if narrator is on the title text is read out a single time. That seems like the right behavior to me.

---
In reply to: [525346775](https://github.com/microsoft/AdaptiveCards/pull/5044#discussion_r525346775) [](ancestors = 525346775,523179028)